app/models/sales_log.rb

```diff
@@ -20,6 +20,7 @@ class SalesLog < Log
   validates_with SalesLogValidator
   before_validation :set_derived_fields!
   before_validation :reset_invalidated_dependent_fields!
+  before_validation :process_mscharge_known_changes!, if: :mscharge_known_changed?
```

Comment:
should we be using a mscharge_known_inferred field like we do for la? or is it fine to infer like this?

Reply:
Do we need to set it to 0? With other fields (for example age fields) we leave it blank

Reply:
Yep, it's specified in the ticket :)

Reply:
Could we add this into the `set_derived_fields!` method instead, cause that's where most stuff like this gets set I think?